ecl/hql/hqlfold.cpp

```diff
@@ -5179,6 +5179,14 @@ IHqlExpression * CExprFolderTransformer::doFoldTransformed(IHqlExpression * unfo
             return removeParentNode(expr);
         }
         break;
+    case no_likely:
+    case no_unlikely:
+        {
+            OwnedHqlExpr folded = transform(expr->queryChild(0));
```

Comment:
This doesn't need to call transform - it has already been called.

    IHqlExpression * child = expr->queryChild(0);